aineko/extras/fastapi/main.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for running a FastAPI server.
+
+This module contains the Consumers and Producers class that give
+access to the node's consumers and producers from within the FastAPI
+app. It also contains the FastAPI node class that runs the uvicorn
+server.
+
+We recommend no more than 1 FastAPI node per pipeline since the Consumer
```

Comment:
I'm glad you addressed this. I was going to ask about to feasibility of running multiple FastAPI nodes in the same pipeline with the new implementation. This seems reasonable.